src/sentry/testutils/helpers/features.py

```diff
@@ -7,7 +7,21 @@
 
 
 @contextmanager
-def Feature(name, active=True):
+def Feature(names):
+    """
+    Control whether a feature is enabled.
+
+    >> with Feature({'feature-1': True, 'feature-2': False}):
+    >>   # Executes with feature-1 enabled and feature-2 disabled
+
+    The following two invocations are equivalent:
+
+    >> with Feature(['feature-1', 'feature-2']):
+    >> with Feature({'feature-1': True, 'feature-2': True}):
```

Comment:
Good docstring

(If you're trying to emulate the interactive shell display, you'd want `>>> ` instead of `>> ` though.)

Reply:
whoops, yup. Though I'd probably want to make it a bit more clear that those are two separate lines :-)